pkgs/applications/graphics/mcomix3/default.nix

```diff
@@ -0,0 +1,85 @@
+{ stdenv
+, fetchFromGitHub
+, python3
+, python3Packages
+, wrapGAppsHook
+, gobject-introspection
+, gtk3
+, gdk-pixbuf
+# Recommended Dependencies:
+, unrar
+, p7zip
+, lhasa
+, mupdf
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "mcomix3";
+  version = "20201123";
+
+  # fetch from github because no official release on pypi/github and no build system
+  src = fetchFromGitHub {
+    repo   = "mcomix3";
+    owner  = "multiSnow";
+    rev = "cdcb27533dc7ee2ebf7b0a8ab5ba10e61c0b8ff8";
+    sha256 = "0q9xgl60ryf7qmy5vgzgfry4rvw5j9rb4d1ilxmpjmvm7dd3fm2k";
+  };
+
+  nativeBuildInputs = [ wrapGAppsHook ];
+  propagatedBuildInputs = (with python3Packages; [ pillow pygobject3 pycairo ])
+    ++ [ gobject-introspection gtk3 gdk-pixbuf ]
+    ++ [ unrar p7zip lhasa mupdf ];
+
+  format = "other";
+  strictDeps = false;
+
+  preInstall = ''
+    libdir=$out/lib/${python3.libPrefix}/site-packages
+    mkdir -p $out/share/{icons/hicolor,man/man1,applications,metainfo,thumbnailers}
+    mkdir -p $out/bin $libdir
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    ${python3.executable} installer.py --srcdir=mcomix --target=$libdir
```

Comment:
Looks like https://github.com/multiSnow/mcomix3/issues/93 still is not resolved :-/

Reply:
Yes, and sadly, it likely never will. So I figured this is better than not having mcomix, wouldn't you agree?